component-runtime-manager/src/test/java/org/talend/sdk/component/runtime/manager/extension/ComponentSchemaEnricherTest.java

```diff
@@ -35,17 +36,28 @@ void fixedSchemaMetadataPresent() {
 
             {
                 put("tcomp::ui::schema::fixed", "discover");
+                put("tcomp::ui::schema::flows::fixed", "main,reject");
```

Comment:
is the default flow is called 'main' ? it is not "__ default __" ? 
https://github.com/Talend/component-runtime/blob/da88e5872597d1dd354d41a468b285b70ba1b3b8/component-api/src/main/java/org/talend/sdk/component/api/processor/Output.java#L28C29-L28C40

Reply:
Yep, `__default__`  is the std name. But, that's a test, it could be _toto_.